repair/row.hh

```diff
@@ -71,6 +71,22 @@ public:
         }
         return _fm->representation().size();
     }
+    size_t external_memory_usage() const {
+        size_t size = 0;
+        if (_fm) {
+            size += _fm->representation().size();
+        }
+        if (_dk_with_hash) {
+            size += _dk_with_hash->dk.external_memory_usage();
+        }
+        if (_boundary) {
+            size += _boundary->pk.external_memory_usage() + _boundary->position.external_memory_usage();
+        }
+        if (_mf) {
+            size += _mf->memory_usage();
+        }
+        return size;
```

Comment:
We can add the sizeof(_hash) to size.

Reply:
std::optional always consumes space, even if it's not engated. We should just add sizeof(repair_row) (but check that the caller doesn't do it already).

Reply:
Yes, indeed. The caller does not add sizeof(repair_row).